### ISSUE TITLE
Remove unwanted python code

### ISSUE THREAD

**Issue by Anish-M-code** [Anish-M-code/Metadata-Remover#7]
Currently the python code in Metadata remover is a huge bloat, it contains lots of code which is never used. Contributors may try to remove as much unwanted code as possible without breaking this project and without removing essential code.

**Comment by Gowrishankar04** [comment]
Hi @Anish-M-code ,

Can you please assign this to me?

Thanks

**Comment by Anish-M-code** [comment]
yeah sure @Gowrishankar04  👍

**Comment by Gowrishankar04** [comment]
Hi @Anish-M-code , Apologies, I couldn't able to resolve this. Can you please unassign from me?

**Comment by Anish-M-code** [comment]
sure @Gowrishankar04 , no problems.

**Comment by ShivamGarg2910** [comment]
@Anish-M-code, assign it to me please.

**Comment by Anish-M-code** [comment]
yeah sure @ShivamGarg2910  👍

**Comment by rohithpeddi7** [comment]
Hi @Anish-M-code  ,

Can you please assign this to me?

I think I found many places where I could modify the code to make it simpler. Thanks!

**Comment by Anish-M-code** [comment]
Are you still interested to work on it @ShivamGarg2910 .

**Comment by PratikLilhare** [comment]
Hi @Anish-M-code, I am looking for my first open-source contribution, would you mind assigning me if the original contributor isn't active?

**Comment by Anish-M-code** [comment]
@PratikLilhare  yeah sure 👍

**Comment by Anish-M-code** [comment]
Are you still interested to work on it @ShivamGarg2910 @PratikLilhare ?

**Comment by PratikLilhare** [comment]
> Are you still interested to work on it @ShivamGarg2910 @PratikLilhare ?

Yes, working on it.

**Comment by Niveditha-K17** [comment]
Hi @Anish-M-code , I'm looking to make my first open source contribution. Is there any issue I can take up?

**Comment by Anish-M-code** [comment]
Yeah sure @Niveditha-K17 , some beginner friendly issues you might consider are:-
https://github.com/Anish-M-code/Metadata-Remover/issues/13
https://github.com/Anish-M-code/File-Analysis-Tool/issues/2
https://github.com/Anish-M-code/pdftotext/issues/3

Please feel free to ask any doubts or questions regarding the code.

**Comment by w4ft** [comment]
It seems that the previous assignees have abandoned this issue, I'd love to take this up!

**Comment by Anish-M-code** [comment]
sure @w4ft you can take this issue if you are interested  👍

**Comment by w4ft** [comment]
@Anish-M-code Great! I was wondering if you guys had any test suites I could use

**Comment by w4ft** [comment]
No worries I can include some in my PR

**Comment by 9x14S** [comment]
Hello, if this position is still open I would like to take it

**Comment by Anish-M-code** [comment]
Yeah sure @9x14S you can give it a try, the issue is still open

**Comment by 9x14S** [comment]
Thanks.

**Comment by gh-adithya** [comment]
Hi! I would like to make my first contribution through this issue. Can I work on it?

**Comment by Anish-M-code** [comment]
@gh-adithya sure

**Comment by Pietrob5** [comment]
Hi! Is this issue still available?

**Comment by RamNisanth** [comment]
Hi is  this issue still available ?

**Comment by Anish-M-code** [comment]
@Pietrob5  @RamNisanth , Sorry for late reply ,Yes this issue is still open , I always try to continuously debloat this application and restructure it any contributions and suggestions are always welcome.

**Comment by RamNisanth** [comment]
let me lookinto it and if I have any queries I will ping here
